### ISSUE TITLE
[BUG] Android App - Zooming in a picture using pinch is almost impossible 

### ISSUE THREAD

**Issue by yurividal** [immich-app/immich#2765]
### The bug

I believe that whatever library you guys are using for the image viewer is way too sensitive to side scrolling.
It's basically impossible to zoom into a picture, it thinks I'm trying to swipe right or left. Double taping works, but only if the 2 taps are EXACTLY precisely on the same spot.
This is a completely different behavior from the phone's gallery and Google photos.

Maybe there is a way to tweak the sensitivity?

### The OS that Immich Server is running on

Ubuntu 22.04 docker

### Version of Immich Server

1.60.0

### Version of Immich Mobile App

1.60.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Same as the example. Works fine.
```

### Your .env content

```Shell
.
```

### Reproduction steps

```bash
1. OPen the app
2. OPen a photo to view.
3. Try pinching to zoom in.
```

### Additional information

Android 13, Samsung Galaxy S23

**Comment by alextran1502** [comment]
Hello would it be possible to have a video that describes the interaction you are experiencing?

**Comment by yurividal** [comment]
> Hello would it be possible to have a video that describes the interaction you are experiencing?

Here it is. Sorry for the terrible quality, had to record it with my crappy laptop camera. But you can clearly see how frustrating it can be. I eventually got it to zoom after a while. But it seems to be very random. Sometimes it zooms just fine, other times it just does this forever.

**Comment by yurividal** [comment]
Sorry, doesn't look like the cideo uploaded. Let me try again.

**Comment by yurividal** [comment]
https://github.com/immich-app/immich/assets/8555749/94ab43ec-4535-4bd2-9b78-bb5608f0dff2

@alextran1502

**Comment by alextran1502** [comment]
Perfect! Thank you

**Comment by Michi1209** [comment]
I don't know if it is related, but the same happens to me sometimes, but only when loading the assets. I have a somewhat old phone, but when the loading icon (top right of the backup) is gone, the zooming works fine. Additionally while loading the app is pretty unresponsive, but this is surely related to my device. Hope this helps.

**Comment by Pheggas** [comment]
It seems to be only occasional issue. I've noticed this a few times but it fixed itself in matter of seconds. It definitely need to look tho.

**Comment by yurividal** [comment]
Yes, it's intermitent indeed. Unfortunately it always happens exactly when I'm trying to show someone a picture, and it makes me look dumb. Hahaha

**Comment by yurividal** [comment]
This is still a problem on the latest version of immich, 1.79.1

There is a fix in the changelog called "Notable fix: Jerky mobile's swipe behavior" but im not sure if it's related.

**Comment by NicholasFlamy** [comment]
> This is still a problem on the latest version of immich, 1.79.1
> 
> There is a fix in the changelog called "Notable fix: Jerky mobile's swipe behavior" but im not sure if it's related.

Check out #5895

**Comment by alextran1502** [comment]
Fixed

**Comment by NicholasFlamy** [comment]
> Fixed

In master or in the latest release?

**Comment by alextran1502** [comment]
Latest release

**Comment by Jozmichne** [comment]
So i am using immich in a humhub installation and i cannot zoom in on the mobile app, will this fix the issue there too? Just run the installer for immich again and it should fix it now?. Sorry I'm new in this ;) apt install command?

**Comment by NicholasFlamy** [comment]
Y'all tweaked that animation length or something. This actually mostly fixes #5895.

**Comment by NicholasFlamy** [comment]
It's like 10x better than it used to be.

**Comment by benjithompson** [comment]
Found this issue (and #5895) after getting frustrated with swipe and zoom behavior. I still think the swipe animation should be much faster. Compared to Photos, the swipe animation seems roughly twice as slow, mostly due to the slow ramp down speed that Immich has.

**Comment by eggermn** [comment]
App version 1.122.2 build 170 on Pixel 8, the bug it's still here.. it's impossible to zoom on motion photo

**Comment by NicholasFlamy** [comment]
> @eggermn we haven't merged and released the fix yet. It was a bug from migrating to the new HDR player

Oh yeah, that's new. But the original issue that this issue is talking about is fixed which I'm ecstatic about.

**Comment by fernfrost** [comment]
This is still an issue in 2.5.6

**Comment by NicholasFlamy** [comment]
> This is still an issue in 2.5.6

You're gonna need to open a new issue and post a video of you demonstrating. Chances are you're touching and dragging with one finger before your other finger touches the screen.
